logger/log.go

```diff
@@ -149,6 +153,23 @@ func init() {
 	defaultLogger = NewLogger(3)
 }
 
+func Cfg(configFile string, debugMod bool, logFile string) {
+	var logCfg logConfig
+	viper.SetConfigType("json")
+	err := viper.Unmarshal(&logCfg)
+	if err != nil {
+		cfg, _ := json.Marshal(&logCfg)
```

Comment:
~/.sealer/config.json is just a string? not a concrete struct? what does the config look like?

Reply:
{"TimeFormat":"2006-01-02 15:04:05","Console":{"level":"","color":true,"LogLevel":5},"File":{"filename":"/root/.sealer/sealer.log","append":true,"maxlines":1000000,"maxsize":1,"daily":false,"maxdays":-1,"level":"TRAC","permit":"0660","LogLevel":0}}
https://github.com/alibaba/sealer/blob/f7218f721e9ca413e1db9a05722a55a72952774b/logger/log_test.go#L56-L70